quickwit/Cargo.toml

```diff
@@ -271,6 +271,7 @@ ulid = "1.1"
 username = "0.2"
 utoipa = { version = "4.2", features = ["time", "ulid"] }
 uuid = { version = "1.10", features = ["v4", "serde"] }
+urlencoding = "2.1.3"
```

Comment:
Please, insert between `ulid` and `username`.

Reply:
the `percent-encoding` also does this